ios/CodePush/CodePush.m

```diff
@@ -7,14 +7,17 @@
 
 #import "CodePush.h"
 
-@interface CodePush () <RCTBridgeModule>
+@interface CodePush () <RCTBridgeModule, RCTFrameUpdateObserver>
 @end
 
 @implementation CodePush {
     BOOL _hasResumeListener;
     BOOL _isFirstRunAfterUpdate;
     int _minimumBackgroundDuration;
     NSDate *_lastResignedDate;
+    long long latestExpectedContentLength;
```

Comment:
Should we prefix these names with underscores? That has been the convention we use for ivars.